docs/pages/cli-docs.mdx

```diff
@@ -870,9 +870,9 @@ $ tctl auth sign --format openssh --host grav-00
 # Invalid command, only one of --user or --host should be set
 $ tctl auth sign --format openssh --host grav-00 --user teleport -o grav_host
 error: --user or --host must be specified
-# create a certificate with a TTL of 10 years for the jenkins user
+# create a certificate with a TTL of 24 hours for the jenkins user
```

Comment:
I would keep this a long lived cert, at least 1y. While we recommend short lived tokens, for new users having longer lived tokens for these static CI/CD are best. We are planning on covering best practices in an upcoming webinar.

Reply:
OK cool. I'd also think 24 hours would be sufficient for any walkthrough. It's less a hard-coded value than an example too. Happy to go with the consensus here. If we want to link to the upcoming best practices and/or Architecture Security article that might be best.

Also, a self-reminder: https://github.com/gravitational/teleport/issues/5163